src/logic/operate.js

```diff
@@ -0,0 +1,17 @@
+import Big from 'big';
+
+export default function operate({ numberOne, numberTwo, operation }) {
+  let result;
+  if (operation === '÷') {
```

Comment:
[OPTIONAL] instead of using Big(numberX) for every operation, you could create a variable and do it once 
var1 = Big(numberOne)
var2 = Big(numberTwo)

then you can use var1 and var2 for other operations

Reply:
Thanks that's a great suggestion! Will do.